Loop/mayoresQueAnterio-A-4.js

```diff
@@ -0,0 +1,25 @@
+// Escriba un programa que pregunte cuántos números se van a introducir, pida esos números, y muestre un mensaje cada vez que un número no sea mayor que el anterior
+const prompt = require("prompt-sync")();
+const amountOfValues = Number(
+	prompt("How many values ​​are you going to enter?: ")
+);
+
+if (amountOfValues < 0) {
+	console.log("Impossible!");
+} else {
+	let anterior;
+	for (i = 0; i < amountOfValues; i++) {
+		if (i === 0) {
+			anterior = Number(prompt("Write a number: "));
+		} else {
+			let valor = Number(
+				prompt(`  Write a number larger than ${anterior} :  `)
```

Comment:
check the spaces, and use the same term to "mayor", it is "greater" change it for "larger" and "higher"